.github/workflows/tests_linux.yml

```diff
@@ -71,18 +71,31 @@ jobs:
               -DENABLE_COVERAGE=ON \
               -DLQ_ENABLE_KERNEL_OMP=ON
 
+            cmake . -BBuildKernelAVXStream -G Ninja \
+              -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
I just noticed that all builds in this action are in debug mode. we may want to consider `RelWithDebInfo` instead. what do you think?

Reply:
Not for coverage

Reply:
doesn't coverage work with DebInfo mode?